src/gui/statusbar.cpp

```diff
@@ -50,8 +50,6 @@
 StatusBar::StatusBar(QStatusBar *bar)
     : m_bar(bar)
 {
-    qApp->setStyleSheet("QStatusBar::item { border-width: 0; }");
```

Comment:
I agree with you about intuitive, but it is blocking dark style sheets - I tested it several times with and without it)